src/gc/impl/conservative/gc.d

```diff
@@ -2854,6 +2854,17 @@ struct Gcx
 
     void scanBackground() nothrow
     {
+        version (linux)
```

Comment:
Is this really specific to linux, or would Posix be the better version?

Reply:
I think it should be valid for any Posix system, although I have access only to linux and OSX.

I don't have such problem with dmd 2.087 under OSX, so I targeted this patch for linux only. I can spend some time to investigate why OSX is not affected.